cumulus_library_data_metrics/c_attachment_count/c_attachment_count.jinja

```diff
@@ -0,0 +1,202 @@
+        THEN 'Anatomic Pathology Structured Report Cancer Testis'
+        WHEN format_code = 'urn:ihe:pat:apsr:cancer:urinary_bladder:2010'
+        THEN 'Anatomic Pathology Structured Report Cancer Urinary Bladder'
+        WHEN format_code = 'urn:ihe:pat:apsr:cancer:lip_oral_cavity:2010'
+        THEN 'Anatomic Pathology Structured Report Cancer Lip Oral Cavity'
+        WHEN format_code = 'urn:ihe:pat:apsr:cancer:pharynx:2010'
+        THEN 'Anatomic Pathology Structured Report Cancer Pharynx'
+        WHEN format_code = 'urn:ihe:pat:apsr:cancer:salivary_gland:2010'
+        THEN 'Anatomic Pathology Structured Report Cancer Salivary Gland'
+        WHEN format_code = 'urn:ihe:pat:apsr:cancer:larynx:2010'
+        THEN 'Anatomic Pathology Structured Report Cancer Larynx'
+        WHEN format_code = 'urn:ihe:pharm:pre:2010'
+        THEN 'Pharmacy Pre'
+        WHEN format_code = 'urn:ihe:pharm:padv:2010'
+        THEN 'Pharmacy PADV'
+        WHEN format_code = 'urn:ihe:pharm:dis:2010'
+        THEN 'Pharmacy DIS'
+        WHEN format_code = 'urn:ihe:pharm:pml:2013'
+        THEN 'Pharmacy PML'
+        WHEN format_code = 'urn:hl7-org:sdwg:ccda-structuredBody:1.1'
+        THEN 'For documents following C-CDA 1.1 constraints using a structured body.'
+        WHEN format_code = 'urn:hl7-org:sdwg:ccda-nonXMLBody:1.1'
+        THEN 'For documents following C-CDA 1.1 constraints using a non structured body.'
+        WHEN format_code = 'urn:hl7-org:sdwg:ccda-structuredBody:2.1'
+        THEN 'For documents following C-CDA 2.1 constraints using a structured body.'
+        WHEN format_code = 'urn:hl7-org:sdwg:ccda-nonXMLBody:2.1'
+        THEN 'For documents following C-CDA 2.1 constraints using a non structured body.'
+        ELSE NULL -- this system binding only holds the above values, ignore unexpected codes
+        END AS display
+
+    FROM attachments
+    -- This system binding is marked as extensible, but for the sake of reliably providing
+    -- consinstent & readable display names, we'll only support the preferred system for now.
+    WHERE format_system = 'http://ihe.net/fhir/ValueSet/IHE.FormatCode.codesystem'
+),
+
+simplified AS (
+    SELECT
+        id,
+        CAST(row AS VARCHAR) AS row,
+        {{ utils.coalesce_missing('status') }} AS status,
+        {{ utils.coalesce_missing('content_type') }} AS content_type,
+        {{ utils.coalesce_missing('language') }} AS language,
+        {{ utils.coalesce_missing('formats.display') }} AS format,
+        CASE
+        WHEN has_data and has_url
+        THEN 'Data and URL'
+        WHEN has_data
+        THEN 'Data'
+        WHEN has_url
+        THEN 'URL'
+        ELSE 'Not available'
+        END AS content
+
+    FROM attachments
+    LEFT JOIN formats USING(id, row)
+)
+
+{% call utils.make_counts('simplified', output_mode, unique_ids=['id', 'row']) %}
+    status,
+    content_type,
+    language,
+    format,
+    content
```

Comment:
Nit, but naming this field `content` seems a little confusing since it doesn't contain the content, though I suppose we could document around that. What about calling it something like `content_location` instead? Similarly, the values of `data` and `url` aren't really self explanatory, and we could consider something like `embedded` and `remote` as alternatives.